finals-2024-01-15/code/5197-elena-dimova/src/app/movie-details/movie.interface.ts

```diff
@@ -0,0 +1,8 @@
+export interface Movie{
```

Comment:
This model does not have the more "interesting" properties (cast, genre, oscars) +2 points